From 10ca5271306182db35c374942ad14d31d53207b7 Mon Sep 17 00:00:00 2001
From: twu14 <tao.t.wu@intel.com>
Date: Wed, 23 Oct 2013 22:12:09 +0800
Subject: Disalbe gcc frame point flag to avoid call stack dump missing.

BZ:181596

Disalbe gcc frame point flag to avoid call stack dump missing.

Signed-off-by: twu14 <tao.t.wu@intel.com>
Change-Id: Iffca7d640ea7eab77a67f3b21a03ae68f67ed507
---
 rogue/build/linux/common/android/arch_common.mk |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/rogue/build/linux/common/android/arch_common.mk b/rogue/build/linux/common/android/arch_common.mk
index 40bc280..ba658b7 100644
--- a/rogue/build/linux/common/android/arch_common.mk
+++ b/rogue/build/linux/common/android/arch_common.mk
@@ -17,6 +17,7 @@ _CLANG	:= $(shell ../tools/cc-check.sh --clang --cc $(_CC))
 SYS_CFLAGS := \
  -fno-short-enums \
  -funwind-tables \
+ -fno-omit-frame-pointer \
  -D__linux__
 SYS_INCLUDES := \
  -isystem $(ANDROID_ROOT)/bionic/libc/arch-$(ANDROID_ARCH)/include \
-- 
1.7.4.1

